### DISCUSSION TITLE
Copilot chat no longer answers based on the context of my code

### DISCUSSION THREAD

**Discussion by cknight** [community/community#108273] [Copilot Conversations]
### Select Topic Area

Bug

### Body

While previously using Copilot chat in VSCode, any question I typed was answered with the context of my currently open file and the surrounding lines of code.  As of the last 24-48 hours, Copilot now just responds with a generic ChatGPT response with no context:

> As an AI, I'm unable to directly view your code....

> As an AI, I don't have access to your specific workspace or files....

I've also tried asking `@workspace Am I rounding correctly on line 89 of islands/export/export.tsx` and get this response:

> I'm sorry, I can't answer that question with what I currently know about your workspace. Please provide the specific code or context from line 89 of islands/export/export.tsx where the rounding operation is being performed.  

With the irony that `islands/export/export.tsx` is hyperlinked to the correct file.

I'm assuming this is a bug with copilot, but if not please explain how I can get the previous behaviour of context based chats.

If it is relevant, I am based in the UK and experienced issues with Copilot yesterday not offering completions.  The completions issue seems to be resolved but have been suffering the chat issue for a few days now.

VSCode:  v 1.86
Copilot extension: v1.162.0
Copilot chat: v0.12.2

**Comment by cknight** [comment]
<img width="412" alt="image" src="https://github.com/community/community/assets/1248238/df5ab524-d453-44dc-9ba9-18e2ef053024">

**Comment by cknight** [comment]
If I highlight the code first and then type  in chat, it will use the highlighted code for reference.  However, it did not previously require this and would work based on open file and cursor position.

**Comment by jtmkrueger** [comment]
I've found that if you want it to look at the current file, you need to prefix the question with `#editor` now. Once you type that `#` into the chat, it'll show you a list of options, in which `#selection` is included. I don't particularly like this new behavior/workflow, but at least the functionality is still there ¯\_(ツ)_/¯

**Comment by cknight** [comment]
This seems to be working again so will close the issue.

**Comment by danailch456** [comment]
I'm still having that issue. Copilot just gives no context answers, it's honestly unusable.

**Comment by Mr-Ruben** [comment]
![https://i.imgur.com/MBiR7n3.png](https://i.imgur.com/MBiR7n3.png)

Happens again in the last version.

```
Version: 1.99.3
Commit: 17baf841131aa23349f217ca7c570c76ee87b957
Date: 2025-04-15T23:18:46.076Z
Electron: 34.3.2
ElectronBuildId: 11161073
Chromium: 132.0.6834.210
Node.js: 20.18.3
V8: 13.2.152.41-electron.0
OS: Linux x64 4.19.0-24-amd64
```


In the previous version I did not have the problem.
